konduit-serving-orchestration/src/main/java/ai/konduit/serving/orchestration/ClusteredInferenceVerticle.java

```diff
@@ -103,7 +103,7 @@ protected void setupWebServer(Promise<Void> startPromise) {
         vertx.createHttpServer()
                 .requestHandler(router)
                 .exceptionHandler(Throwable::printStackTrace)
-                .listen(port, inferenceConfiguration.getServingConfig().getListenHost(), handler -> {
+                .listen(port, inferenceConfiguration.getServingConfig().listenHost(), handler -> {
```

Comment:
Again, konduit-serving-orchestration is an old API module, so this shouldn't be modified.